test/TwigEnvironmentFactoryTest.php

```diff
@@ -19,59 +19,55 @@
 use Mezzio\Twig\TwigEnvironmentFactory;
 use Mezzio\Twig\TwigExtension;
 use Mezzio\Twig\TwigExtensionFactory;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
-use Prophecy\Prophecy\ProphecyInterface;
 use Psr\Container\ContainerInterface;
+use ReflectionException;
+use ReflectionProperty;
 use Twig\Environment as TwigEnvironment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
 use Twig\Extension\CoreExtension;
 use Twig\Extension\EscaperExtension;
 use Twig\Extension\OptimizerExtension;
 use Twig\RuntimeLoader\RuntimeLoaderInterface;
 
-use function is_string;
-
 class TwigEnvironmentFactoryTest extends TestCase
 {
-    /**
-     * @var ContainerInterface|ProphecyInterface
-     */
+    /** @var MockObject<ContainerInterface> */
     private $container;
 
-    protected function setUp() : void
+    protected function setUp(): void
     {
-        $this->container = $this->prophesize(ContainerInterface::class);
+        $this->container = $this->createMock(ContainerInterface::class);
     }
 
-    public function testCallingFactoryWithNoConfigReturnsTwigEnvironmentInstance()
+    /**
+     * @throws LoaderError
```

Comment:
We don't annotate throws on tests. It is not useful, just some extra noise

Reply:
removed all @throws additions in tests